### PR TITLE
Hypo description fix

### PR DESCRIPTION
**Pull request by TGCMbetcher**
## About The Pull Request

Fixes a typo with hyposprays having AN instead of A in the description. Sorry for not combining the two, it was brought up after I finished the other one.

## Why It's Good For The Game

Quick spell fix

## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -288,29 +288,29 @@
 
 /obj/item/reagent_containers/hypospray/advanced/meraderm
 	name = "A meraderm hypospray"
```
Comment (Chaznoodles) [inline code/game/objects/items/reagent_containers/hypospray.dm:290]: Should just be 'Meraderm hypospray'

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -288,29 +288,29 @@
...
-	desc = "An hypospray loaded with meralyne and dermaline."
+	desc = "A hypospray loaded with meralyne and dermaline."
 	list_reagents = list(
 		/datum/reagent/medicine/meralyne = 30,
 		/datum/reagent/medicine/dermaline = 30,
 	)
 
 /obj/item/reagent_containers/hypospray/advanced/meralyne
 	name = "A meralyne hypospray"
```
Comment (Chaznoodles) [inline code/game/objects/items/reagent_containers/hypospray.dm:298]: Same

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -288,29 +288,29 @@
...
-	desc = "An hypospray loaded with meralyne."
+	desc = "A hypospray loaded with meralyne."
 	list_reagents = list(
 		/datum/reagent/medicine/meralyne = 60,
 	)
 
 /obj/item/reagent_containers/hypospray/advanced/dermaline
 	name = "A dermaline hypospray"
```
Comment (Chaznoodles) [inline code/game/objects/items/reagent_containers/hypospray.dm:305]: Same

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -288,29 +288,29 @@
...
-	desc = "An hypospray loaded with dermaline."
+	desc = "A hypospray loaded with dermaline."
 	list_reagents = list(
 		/datum/reagent/medicine/dermaline = 60,
 	)
 	
 /obj/item/reagent_containers/hypospray/advanced/ironsugar
 	name = "A ironsugar hypospray"
```
Comment (Chaznoodles) [inline code/game/objects/items/reagent_containers/hypospray.dm:312]: Same

**Review by Chaznoodles — COMMENTED** [review]
The actual thing I meant to be corrected, but the 'an' is a good spot.

File: code/modules/modular_armor/attachables.dm [hunk 5]
```diff
@@ -50,7 +50,7 @@
 
 /obj/item/armor_module/attachable/better_shoulder_lamp/mark1
 	name = "\improper Mark 1 Baldur Light Amplification System"
-	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
```
Comment (TiviPlus) [inline code/modules/modular_armor/attachables.dm:53]: unrelated changes

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/modular_armor/attachables.dm (hunk 5 above)
Comment (TGCMbetcher) [reply code/modules/modular_armor/attachables.dm:53]: Yes its the other one put together. I dont know how much a single addition hurts the total so I can close the other one if you want and keep them together. Title the whole thing description fixes.
